speechbrain/nnet/ema.py

```diff
@@ -0,0 +1,74 @@
+"""
+Library implementing learnable exponential moving average layer
+in compliance with LEAF
+
+Neil Zeghidour, Olivier Teboul, F{\'e}lix de Chaumont Quitry & Marco Tagliasacchi, "LEAF: A LEARNABLE FRONTEND
+FOR AUDIO CLASSIFICATION", in Proc of ICLR 2021 (https://arxiv.org/abs/2101.08596)
+
+Authors
+ * Sarthak Yadav 2022
+"""
+import torch
+from torch import nn
+
+
+class ExponentialMovingAverage(nn.Module):
```

Comment:
Shouldn't this go to speechbrain.nnet.normalization? That is literally a question ahah. The idea always is to reduce the number of files.

Reply:
Well the idea was that EMA might find other use cases. But I'll move it to speechbrain.nnet.normalization, it goes well there too.